tests/monitoring_test.go

```diff
@@ -108,7 +108,7 @@ var _ = Describe("Prometheus Alerts", func() {
 			template *templatev1.Template
 		)
 		BeforeEach(func() {
-			template = TemplateWithRules()
+			template = TemplateWithRules("q35")
```

Comment:
Yes, we can. I was considering creating a new pull request to make the method available for both monitoring_tests and validator_tests. This approach will help keep the pull request cleaner. What do you think?

Reply:
Yeah sure. You would do it in this same PR, as you prefer.

Reply:
I experienced a déjà vu and realized that my previous comment didn’t make much sense :laughing: 
If monitoring_test.go can access `TemplateWithRules`, it can also access `retrieveQemuMachineType`. I’ll make the necessary changes, and I apologize for the confusion.
Regardless, the test passes since the type is not relevant to the test. It’s also a bit confusing that there is only one test, yet it uses both `BeforeEach` and `AfterEach`.